aries-site/src/pages/templates/wizard.mdx

```diff
@@ -73,6 +73,24 @@ If guidance can be provided in a short description, use a single column wizard.
 - On desktop, place the guidance to the right of form inputs.
 - On mobile, place the guidance above form inputs.
 
+## Alignment
+
+### Previous step button
+
+A button that enables the user to navigate to the previous step should be placed in the wizard header and aligned to the far-left side of the header.
+
+### Cancel button
+
+A button that enables the user to cancel out of the wizard should be placed in the wizard header and aligned to the far-right side of the header.
```

Comment:
is Wizard capital?

Reply:
I'm never sure when to capitalize or not, but I think because this is just referring more generically to the concept of a wizard (not a formalized component) then it's okay to be lower case.

Reply:
okay fair also not exactly sure.